getcache.go

```diff
@@ -71,6 +71,10 @@ func (b *buildCache) GetWithRemoteAPI(ctx context.Context, image string) (io.Rea
 	if versions.LessThan(v.Version, "1.12.0") {
 		logrus.Warnf("Docker versions before v1.12.0 have a bug causing extracting build cache through remote API to take very long time and use lots of disk space. Please consider upgrading before using this tool.")
 	}
+	
+	if versions.GreaterThanOrEqualTo(v.Version, "1.12.2") {
```

Comment:
No, it's only in master builds and will be released in `1.13`. `1.12.2` is a patch release with only bugfixes and no new features.